--- doc/sample-ngircd.conf.tmpl	2011-06-28 12:50:11.000000000 +0200
+++ doc/sample-ngircd.conf.tmpl.oden	2011-08-24 11:13:23.991808254 +0200
@@ -191,13 +191,13 @@
 	# So don't forget to remove the ";" above if this is the case ...
 
 	# SSL Server Key Certificate
-	;CertFile = :ETCDIR:/ssl/server-cert.pem
+	;CertFile = :ETCDIR:/pki/tls/certs/ngircd.pem
 
 	# Diffie-Hellman parameters
-	;DHFile = :ETCDIR:/ssl/dhparams.pem
+	;DHFile = :ETCDIR:/pki/tls/private/ngircd-dhparams.pem
 
 	# SSL Server Key
-	;KeyFile = :ETCDIR:/ssl/server-key.pem
+	;KeyFile = :ETCDIR:/pki/tls/private/ngircd.pem
 
 	# password to decrypt SSLKeyFile (OpenSSL only)
 	;KeyFilePassword = secret
